### DISCUSSION TITLE
Docker release?

### DISCUSSION THREAD

**Discussion by ProvoK** [emad-elsaid/xlog#18] [Ideas]
For my use-case I will build a Docker image for this.

I can easily open a PR for that, I believe it would be beneficial.
Given that everything is stored on disk, I believe there will not issue of any sort.

What do you think?

**Comment by emad-elsaid** [comment]
Good idea, I'll review the PR as soon as it's opened.

**Comment by emad-elsaid** [comment]
Done #20
